cardano-db-sync/src/Cardano/DbSync/LedgerState.hs

```diff
@@ -305,6 +335,40 @@ ledgerRewardUpdate env lsc =
       LedgerStateAllegra als -> Generic.allegraRewards env als
       LedgerStateMary mls -> Generic.maryRewards env mls
 
+-- Given an address, return it's current UTxO balance.
+ledgerAddrBalance :: Text -> LedgerState (CardanoBlock StandardCrypto) -> Either Text Word64
```

Comment:
If the ledger state was validated in `cardano-db-tool` then then the following could be in the `cardano-db/app/Cardano/Db/App` part of the tree.

Reply:
moved to `cardano-db-tool/src/Cardano/Db/Tool/Validate/Balance.hs`